scripts/ci-build-and-test

```diff
@@ -109,28 +113,29 @@ ccache -s
 mv test-install test-install-relocated || exit $?
 
 set +xv
-printf "calling ${PWD}/test-instal-relocatedl/bin/kim-api-activate\n"
+printf "calling ${PWD}/test-install-relocated/bin/kim-api-activate\n"
 source "${PWD}/test-install-relocated/bin/kim-api-activate" || exit $?
 set -xv
 kim-api-collections-management install system `find ../examples/model-drivers -mindepth 1 -maxdepth 1 -type d` &&
   kim-api-collections-management install system `find ../examples/portable-models -mindepth 1 -maxdepth 1 -type d` &&
   kim-api-collections-management install system `find ../examples/simulator-models -mindepth 1 -maxdepth 1 -type d` &&
   kim-api-collections-management list &&
-  ./test-install-relocated/libexec/kim-api/kim-api-simulator-model Sim_LAMMPS_LJcut_AkersonElliott_Alchemy_PbAu smspec-file data \
+  kim-api-simulator-model Sim_LAMMPS_LJcut_AkersonElliott_Alchemy_PbAu smspec-file data \
```

Comment:
This won't work on linux/mac since `.../libexec/...` is not on the path

Reply:
Then we might have to use if/else for conditional execution of this statement on Windows. On the Windows platform, the executable `kim-api-simulator-model` gets installed in `bin/`, not `libexec/kim-api/`.